src/core/viz/expressions/globalAggregation.js

```diff
@@ -7,22 +7,22 @@ import { implicitCast } from './utils';
 
 
 /**
- * Return the average value of all the features.
+ * Return the average of the feature property for the entire source.
```

Comment:
I find "entire source" less clear than "all the features"... perhaps "all the source's features"? ... "the entire source data"?